### PR TITLE
Remove duplicates in Linked List using Hashing

### PR DESCRIPTION
**Pull request by Sarthak-9**
Issue #1891 
Remove Duplicates from an Unsorted Linked List using hashing in C++.

### CODE REVIEW

**Comment by Sarthak-9** [comment]
@Rohan27s Please review.

**Comment by Sarthak-9** [comment]
@nakul-19 Please check.

**Comment by Sarthak-9** [comment]
@tarun26091999 No updates yet

File: Code/C++/remove_duplicates.cpp [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+/*
+
+ Problem : Remove duplicates from an unsorted linked list
```
Comment (Rohan27s) [inline Code/C++/remove_duplicates.cpp:6]: Mention this statement above the code

File: Code/C++/remove_duplicates.cpp [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+ Algorithm : This problem can be solved by hashing. The elements are hashed to a hash array and elements with frequency more than 1 are picked only once.
+ Time Complexity : O(n)
```
Comment (Rohan27s) [inline Code/C++/remove_duplicates.cpp:8]: Provide the time and space complexity at the end of the code(after the test cases), and also mention that what "n" stands for.

**Review by Rohan27s — CHANGES_REQUESTED** [review]

File: Code/C++/remove_duplicates.cpp (hunk 2 above)
Comment (Sarthak-9) [reply Code/C++/remove_duplicates.cpp]: @Rohan27s Check now

**Review by Rohan27s — APPROVED** [review]

**Pull request merged**